src/pages/Admin/Venue/Details/VenueDetails.tsx

```diff
@@ -0,0 +1,254 @@
+  };
+
+  const handleEditRoomSave = (values: RoomData_v2, index: number) => {
+    const newData = {
+      ...values,
+      x_percent: editingRoom?.x_percent,
+      y_percent: editingRoom?.y_percent,
+      width_percent: editingRoom?.width_percent,
+      height_percent: editingRoom?.height_percent,
+    };
+
+    updateRoom(newData, venueId!, user, index);
+  };
+
+  return (
+    <S.Container>
+      <S.Header>
+        <VenueHero
+          bannerImageUrl={coverImageUrl}
+          logoImageUrl={icon}
+          name={name}
+          subtitle={subtitle}
+          description={description}
+          large
+          showEdit
+          venueId={venue.id!}
+        />
+
+        <S.HeaderActions>
+          <Link
+            to={`/in/${venue.id}`}
+            className="btn btn-primary"
+            style={{ marginBottom: "0.5em" }}
+            target="_blank"
+          >
+            Go to your space
+          </Link>
+          <Button>Preview landing page</Button>
+
+          <S.AdminList>
+            <S.AdminListTitle>Party admins</S.AdminListTitle>
+
+            {ownersData.length > 0 &&
+              ownersData.map((owner: Owner) => (
+                <S.AdminItem key={owner.id}>
+                  <S.AdminPicture backgroundImage={owner.pictureUrl} />
+                  <S.AdminItemTitle>{owner.partyName}</S.AdminItemTitle>
+                </S.AdminItem>
+              ))}
+
+            <Button onClick={() => setShowOwnersModal(true)}>
+              Invite an admin
+            </Button>
+          </S.AdminList>
+        </S.HeaderActions>
+      </S.Header>
+
+      <S.Main>
+        <MapPreview
+          venueId={venueId!}
+          venueName={name}
+          mapBackground={mapBackgroundImageUrl}
+          rooms={rooms}
+        />
+
+        {!!mapBackgroundImageUrl && (
+          <>
+            <S.RoomActions>
+              <S.RoomCounter>{rooms ? rooms.length : "0"} Rooms</S.RoomCounter>
+              <Button onClick={() => toggleRoomModal()} gradient>
+                Add a room
+              </Button>
+            </S.RoomActions>
+          </>
+        )}
+
+        {!!rooms && !!mapBackgroundImageUrl && (
+          <S.RoomWrapper>
+            {rooms.map((room: AnyRoom, index: number) => (
+              <RoomCard
+                key={room.title}
+                room={room}
+                venueId={venueId!}
+                editHandler={() => handleEditRoom(room, index)}
+                onEventHandler={handleRoomEvent}
+                roomIndex={index}
+              />
+            ))}
+          </S.RoomWrapper>
+        )}
+      </S.Main>
+
+      <RoomModal
+        isVisible={modalOpen}
+        venueId={venueId!}
+        onSubmitHandler={() => setModalOpen(false)}
+        onClickOutsideHandler={() => setModalOpen(false)}
+      />
+
+      {editingRoom && (
+        <RoomEdit
+          isVisible={!!editingRoom}
+          onClickOutsideHandler={() => setEditingRoom(null)}
```

Comment:
.

Reply:
Maybe call this `cancelRoomEditing`/`cancelEditingRoom` or something like that?